workflow-templates/phpunit-sqlite.yml

```diff
@@ -89,7 +98,9 @@ jobs:
         # Only run if phpunit config file exists
         if: steps.check_composer.outputs.files_exists == 'true'
         working-directory: apps/${{ env.APP_NAME }}
-        run: composer i
+        run: |
+          composer remove nextcloud/ocp --dev
```

Comment:
what we usually do there is require the oldest version of nextcloud/ocp, that is supported by the app, in composer.json:
https://github.com/nextcloud/announcementcenter/blob/main/composer.json#L13

and install the other versions on top for psalm:
https://github.com/nextcloud/announcementcenter/blob/main/.github/workflows/psalm-matrix.yml#L27

ignoring the platform dependencies:
https://github.com/nextcloud/announcementcenter/blob/main/.github/workflows/psalm-matrix.yml#L47

Reply:
ok, removed this change